docs/index.rst

```diff
@@ -49,32 +49,53 @@ Questions can also be asked on
 Some recent development changelog can be seen `here <https://github.com/rwth-i6/returnn/blob/master/CHANGELOG.md>`__.
 
 
-User guide
-----------
+.. Getting Started
+.. ---------------
 
 .. toctree::
+    :hidden:
+    :caption: Getting Started
+    :maxdepth: 2
+
+    getting_started/installation
+    getting_started/basic_usage/basic_usage
+    getting_started/framework.rst
+    getting_started/faq.rst
+
+.. toctree::
+    :hidden:
+    :caption: Reference
     :maxdepth: 2
 
-    installation
-    basic_usage
-    tech_overview
-    tf_lstm_benchmark
-    multi_gpu
-    deterministic_training
-    debugging
-    profiling
     configuration_reference/index.rst
     dataset_reference/index.rst
     layer_reference/index.rst
 
+.. toctree::
+    :hidden:
+    :caption: Advanced Topics
+    :maxdepth: 2
+
+    advanced/pretraining.rst
+    advanced/multi_gpu.rst
+    advanced/debugging.rst
+    advanced/profiling.rst
+    advanced/deterministic_training.rst
 
-API Reference
--------------
+.. toctree::
+    :hidden:
+    :caption: Applications
+
+    applications/asr.rst
+    applications/lm.rst
+    applications/mt.rst
 
 .. toctree::
-    :maxdepth: 2
-    :glob:
+    :hidden:
+    :caption: Internals
 
+    internals/tech_overview.rst
```

Comment:
I'm not sure I would put this to the very end. Rather, I think that should be almost as the very front, as this is one of the most important page for newcomers, to understand how RETURNN works (after maybe reading the basic usage page).

Reply:
I am not sure I can agree on this one, as for me this page mostly presents Information that is needed when you start making code changes to RETURNN, rather than using RETURNN.

Reply:
But I would argue that you should have a good overview of how RETURNN works internally to really work with RETURNN. A lot of questions and problems just arise because people never really understood how RETURNN is designed.

Reply:
Is it correct that `tech_overview` is both under `internals` and `getting_started` now?

Reply:
Ping here. (Maybe you missed it?)